tunnel/intra.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	"github.com/Jigsaw-Code/outline-go-tun2socks/tunnel/intra"
 	"github.com/Jigsaw-Code/outline-go-tun2socks/tunnel/intra/doh"
+	"github.com/Jigsaw-Code/outline-go-tun2socks/tunnel/intra/protect"
```

Comment:
This code and doh.go don't need to be aware of the protector. That's an artifact of it being used in a VPN context.

Instead of passing a protector, you can pass the dialer or listenconfig instead.

Reply:
Done!